test/test_torch.py

```diff
@@ -8499,6 +8505,31 @@ def test_multidim(x, dim):
             expected = fn(y, 1, keepdim=False)
             self.assertEqual(x[:, 1], expected, '{} with out= kwarg'.format(fn_name))
 
+    @onlyCPU
+    @dtypes(torch.bfloat16)
+    def test_std_var(self, device, dtype):
```

Comment:
This test is copypasted from test_reduction_empty and it does not test the actual operation. If you feel test_reduction_empty should be extended, please do that. Don't copy paste things.